Fix compilation on iPad Pro 1st Gen with iPadOS13 and iOS 12

fabstu opened the pull request:
Fixes #124 by making it compile again on said platforms.

@Lukasa is this the correct fix?

Lukasa commented on Tests/NIOSSLTests/NIOSSLIntegrationTest.swift:
```diff
@@ -21,7 +21,7 @@ import CNIOBoringSSL
 @testable import NIOSSL
 import NIOTLS
-import class Foundation.Process
+//import class Foundation.Process
```
I think commenting this out is unwise: if we don’t need it, we should delete it.

fabstu commented:
Good to know I didn't realize. I removed my header-changes and then added it to `scripts/patch-2-arm-arch.patch` and regenerated the headers. I removed the commented line too.

Lukasa commented:
I *think* it’s correct, but I’m away from my computer right now and need to double check. Does the branch now build on your iPad?

fabstu commented:
Yes it does. Didn't expect it to be that easy to fix. Such small things can be quite annoying sometimes.

Lukasa commented:
@swift-nio-bot Test this please

Lukasa commented:
Great, shipped this fix in 2.3.1 which should unblock your issues. Thanks for contributing the patch! :sparkles:
